modules/standard/Random.chpl

```diff
@@ -135,13 +288,15 @@ class RandomStream {
     return result;
   }
 
-  proc fillRandom(X: [], param parSafe = this.parSafe) {
-    if X.eltType != complex && X.eltType != real && X.eltType != imag then
+  pragma "no doc"
+  proc fillRandom(arr: [], param parSafe = this.parSafe) {
```

Comment:
I was thinking that that iterator didn't work as a standalone call due to leader-follower limitations, but was mistaken.  Added a description of it in my next draft.